OrchardCore.Commerce.Tests/MoneyServiceTests.cs

```diff
@@ -71,17 +61,20 @@ public void UnknownCurrencyCodeGivesNullResult()
         [Fact]
         public void CreateMakesAmountWithCurrency()
         {
-            Amount amount = new TestMoneyService().Create(42, "AMD");
+            var service = new TestMoneyService();
+            var amount = service.Create(42, "AMD");
+
             Assert.Equal(42, amount.Value);
-            Assert.Equal(AnkhMorporkDollar, amount.Currency);
+            Assert.Equal(service.GetCurrency("AMD"), amount.Currency);
```

Comment:
Why this change?

Reply:
i fails.  AMD is a valid currency Armenian Dracma or something and it would always be picked first.

Reply:
I meant why remove `AnkhMorporkDollar` and change its ISO symbol to something unique?

Reply:
Not sure what this "feature" is for, it allows external parties to inject currencies using a means other than through registered providers.  Where is this rouge currency meant to be saved?  Can you please explain the intent.

Reply:
why do we want to allow currencies to be injected when we have providers?  any external source can then create any currency?  Where should they be stored?  A new user provider maybe?

Reply:
I'm very confused by those questions. I don't know what feature you're referring to, what you mean by inject in this context, or what you mean by "saved".

Here's an example of a currency being created outside of the scope of any provider: deserialization (providers are not available in this context, so we have to be able to create a currency from ISO code alone).